hw/ip/flash_ctrl/data/flash_ctrl.hjson.tpl

```diff
@@ -1132,24 +1142,32 @@
               '''
           },
           { bits: "3",
+            name: "lcmgr_intg_err",
```

Comment:
Mostly a reminder to myself: I should update the DIF to reflect the new bits.

Reply:
o i was going to do that after this PR :)